packages/lesswrong/server/repos/PostViewTimesRepo.ts

```diff
@@ -17,10 +17,10 @@ class PostViewTimesRepo extends IncrementalViewRepo<"PostViewTimes"> {
   }: {
     startDate: Date;
     endDate: Date;
-  }): Promise<Omit<DbPostViewTime, "_id" | "schemaVersion" | "createdAt" | "legacyData">[]> {
+  }): Promise<IncrementalViewDataType<"PostViewTimes">[]> {
     // Round startDate (endDate) down (up) to the nearest UTC date boundary
-    const windowStart = moment.utc(startDate).startOf("day").toDate();
-    const windowEnd = moment.utc(endDate).endOf("day").toDate();
+    const startDateFormatted = moment.utc(startDate).startOf("day").format('YYYY-MM-DD HH:mm:ss');
+    const endDateFormatted = moment.utc(endDate).endOf("day").format('YYYY-MM-DD HH:mm:ss');
```

Comment:
This is essentially the fix, I am just turning everything into strings to be extra careful about the timezones

Reply:
Yep, and then the timezone got removed when they were used as parameters below. I think the essential "root cause" of this issue is that the `timestamp` column on the analytics table has type `timestamp without timezone`

Reply:
Oh! yeah seems bad